features/polyExplorer/src/locales/de/stories/clusterMessengerStory.json

```diff
@@ -19,6 +19,8 @@
     "overview.donut.users.message": "Monatlich aktive Nutzer in Millionen",
     "details.section": "Details",
     "details.p.1": "Today our digital lives are all about being connected. Messenger apps play a key role in that. They shape our communication and our relationships. Many of us use those apps very, very often. A lot of information is sent and received. Some of that information might be personal or even secret.",
+    "details.monthly.active.users": "Monatliche aktive Nutzer (MAN in Millionen)",
```

Comment:
Never heard `MAN` before for `MAU`, but if that's how people want it we can keep it :D

Reply:
It was improvised to be honest :D Monatliche Aktive Nutzer or do we keep the englisch acronym here?